can/bus.py

```diff
@@ -152,13 +152,37 @@ def _recv_internal(
         """
         raise NotImplementedError("Trying to read from a write only bus?")
 
+    def send(
+        self, msgs: Union[Sequence[Message], Message], timeout: Optional[float] = None
+    ):
+        """Transmit a message to the CAN bus.
+
+        Override this method to enable the transmit path.
+
+        :param msgs: A message object of sequence of message objects.
+
+        :param timeout:
+            If > 0, wait up to this many seconds for message to be ACK'ed or
+            for transmit queue to be ready depending on driver implementation.
+            If timeout is exceeded, an exception will be raised.
+            Might not be supported by all interfaces.
+            None blocks indefinitely.
+
+        :raises can.CanError:
+            if the message could not be sent
+        """
+        for msg in self._get_messages_sequence(msgs):
+            self._send_internal(msg, timeout)
+
     @abstractmethod
-    def send(self, msg: Message, timeout: Optional[float] = None):
+    def _send_internal(
+        self, msgs: Union[Sequence[Message], Message], timeout: Optional[float] = None
+    ):
         """Transmit a message to the CAN bus.
 
         Override this method to enable the transmit path.
```

Comment:
This comment can now be updated to be user facing, and we will have to update the [creating a new interface docs](https://python-can.readthedocs.io/en/master/development.html#creating-a-new-interface-backend)